reward-call.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Reward Calls
+sidebar_position: 6
+---
+
+This guide provides instructions and recommendations on the ways to call reward when [activated](/video-miners/getting-started/activation) on the Livepeer network. It is assumed you have prior knowledge about [Earnings](/video-miners/core-concepts/earnings#rewards) and that your orchestrator is [activated](/video-miners/getting-started/activation).
+
+
+# About Calling Reward
+
+By default, an active orchestrator will automatically call reward in each round, submitting an Arbitrum transaction that distributes newly minted LPT rewards to itself and its delegators.
+
+The amount of LPT rewards distributed by the reward call depends on the orchestrator's stake, i.e. its own stake and that of its delegators. It is important to note that for orchestrators with very low stake, the ETH transaction cost of calling reward may exceed the amount of LPT rewards distributed.
+
+## Getting Started with Reward Calls
+
+When you first initiate reward calls, it may make economic sense you [disable automatic reward calls](/video-miners/getting-started/reward-call#disable-automatic-reward-calls) and then [manually call reward](/video-miners/getting-started/reward-call#manually-call-reward) in each round instead. 
+
+You then can [enable automatic reward calls](/video-miners/getting-started/reward-call#enable-automatic-reward-calls) when you are confident the distribution of LPT relative to the ETH transaction cost makes economic sense.

```

Comment:
`confident the distribution` -> `confident *that* the distribution`

might just be a stylistic thing

Reply:
not necessary, but if it seems better to you I can make the change